apps/passport-server/src/services/e2eeService.ts

```diff
@@ -57,6 +60,61 @@ export class E2EEService {
 
     res.json(undefined satisfies UploadEncryptedStorageResponseValue);
   }
+
+  public async handleUpdate(
```

Comment:
you've got inconsistent terminology - `handleUpdate` vs `changeBlobKey` - i like the 2nd a lot more